testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSWalletTest.scala

```diff
@@ -686,4 +695,67 @@ object BitcoinSWalletTest extends WalletLogger {
     } yield ()
   }
 
+  /** Constructs callbacks for the wallet from the node to process blocks and compact filters */
+  def createNeutrinoNodeCallbacksForWallet(wallet: Wallet)(implicit
```

Comment:
Perhaps in the future we want to use the callbacks defined in 

https://github.com/bitcoin-s/bitcoin-s/blob/4e862b7297668272933101d73b6867dd211ea026/app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala#L197